ee/desktop/runner/runner.go

```diff
@@ -145,15 +157,16 @@ func (r *DesktopUsersProcessesRunner) Execute() error {
 		}
 	}
 
-	f()
-
 	ticker := time.NewTicker(r.updateInterval)
 	defer ticker.Stop()
 
 	for {
+		// Check immediately on each iteration, avoiding the initial ticker delay
+		f()
```

Comment:
this pattern is better

is there any reason to keep the function as a variable?

Reply:
Nope - simplified this and got rid of the var.